pages/details/[id].js

```diff
@@ -51,6 +52,21 @@ export default function Details({ id, destinationDB, toDosDB }) {
 
   const toggleShowProfile = () => setShowProfile((showProfile) => !showProfile);
   const toggleLoader = () => setLoader((loader) => !loader);
+  const toggleModal = (modalName = "", type = "", id = "") =>
+    setModal((modal) => ({ visible: !modal.visible, name: modalName, type: type, id: id }));
```

Comment:
I'm wondering, what do you need the `id` for here? I'm not critiquing, I'm just not seeing how the id is doing anything.

Reply:
Man you have eagle eyes :) i removed the id everywhere since i never used it. Thanks!